sklearn/utils/extmath.py

```diff
@@ -599,8 +599,10 @@ def log_logistic(X, out=None):
     http://fa.bianp.net/blog/2013/numerical-optimizers-for-logistic-regression/
     """
     is_1d = X.ndim == 1
+    X = np.atleast_2d(X)
     X = check_array(X, dtype=np.float)
```

Comment:
Not related to this PR but this should be changed to `dtype=np.float64` to be explicitly not platform dependent.